inst/views/components/metrics_charts.html

```diff
@@ -201,10 +194,20 @@
 
   // tweak chart y-axis
   this.$adjust_axes = function(chart, metric, data) {
+
+    // adjust x-axis range
     var total_epochs = this.$get_total_epochs(data);
     chart.axis.max({
       x: total_epochs
     });
+
+    // define new x axis tick generator
+    var self = this;
```

Comment:
We have a `thiz` variable already defined for this case so you should just be able to use that.

Reply:
Thanks! I missed that.